charts/egeria-platform/values.yaml

```diff
@@ -0,0 +1,117 @@
+# Egeria Settings
+egeria:
+  # Global Egeria Settings
+  logging: error
+  version: "3.5"
+  debug: true
+
+  # persistence
+  persistence: true
+  storageClass: ibmc-vpc-block-10iops-tier
+  storageSize: 8Gi
+  repositoryType: "local-graph-repository"
+
+  # communication
+  cohort: mds
+  viewOrg: org
+  kafkaEndpoint: egeria-kafka:9092
+
+  # userid used for admin ops (no security enabled by default)
+  user: garygeeke
+
+  # list of servers that should start on this platform
+  # the names must correspond to a config-<name>.sh script
+  # as well as to the actual instance name
+  serverList: mds1,view1
+
+  # OMAG Server
+  # usually you need to configure a connection to an OMAG server
+  # the values below will result in EGERIA_OMAG_SERVER_NAME and EGERIA_OMAG_SERVER_URL environment variables
+  # if platformUrl is empty this will point to itself
+  omagServer:
```

Comment:
the omagserver is really only needed for the polymer UI, which we aren't using (this may have been in source chart too). There also was a reference in the view server config, but none of the current view servers use this afaik.

Reply:
I think this should be removed as confusing to the chart user?